### DISCUSSION TITLE
How to set the dpi as part of the resize process?

### DISCUSSION THREAD

**Discussion by littlegreendude** [dlemstra/Magick.NET#1351] [Help] [answered]
Is it possible to take a 5 x 7" png @ 96dpi and resize it to 5 x 7" @300 dpi?

**Comment by dlemstra** [comment, marked as answer]
That would mean you would need to upscale the image because an image that is 5 x 7" @300 dpi needs more pixels than 5 x 7" png @ 96dpi.

**Reply by littlegreendude** [reply]
Yes, and is there a method to do that?

**Reply by littlegreendude** [reply]
I think there may be a way to do it with resample, but I'm having difficulties tracking down a good example.

**Reply by dlemstra** [reply]
The quality of your output image might be worse but with 5 x 7" @ 96dpi you have an image that is 480 x 672 pixels. And 5 x 7" @300 dpi is an image that is 1500 x 2100 pixels. So you will need to `.Resize()` your image to that dimension and then set the `.Density` property of the image.

**Comment by littlegreendude** [comment]
I don't know if it matters, but my starting image is 20 x 30 inches @96 dpi?

**Reply by littlegreendude** [reply]
That worked.  Thank you.
